src/collection-facets/facets-template.ts

```diff
@@ -8,77 +8,21 @@ import {
 } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 import { repeat } from 'lit/directives/repeat.js';
-import {
-  FacetGroup,
-  FacetBucket,
-  SelectedFacets,
-  getDefaultSelectedFacets,
-  FacetEventDetails,
-} from '../models';
+import type { FacetGroup, FacetBucket, FacetEventDetails } from '../models';
 import type { CollectionTitles } from '../data-source/models';
-import { FacetRow } from './facet-row';
+import './facet-row';
 
 @customElement('facets-template')
 export class FacetsTemplate extends LitElement {
   @property({ type: Object }) facetGroup?: FacetGroup;
 
-  @property({ type: Object }) selectedFacets?: SelectedFacets;
-
-  @property({ type: String }) renderOn?: string;
```

Comment:
I've sort of gutted this component in order to eliminate some anti-patterns:

1. Inverted update flow (`selectedFacetsChanged` etc). This component's prior behavior was to accept the `selectedFacets` state, modify that state non-immutably, and then emit an event notifying parents of the state change. This felt somewhat backwards. The best practice we aim to follow is pushing state changes downwards, not upwards -- low-level components should emit events that allow their ancestors to update the state appropriately and flow any changes back down. So, I've migrated the parents to listen for the `facetClick` event instead, eliminating any need for `selectedFacets` here because this component has no business modifying that high-level state directly.

3. Explicitly caring where it's rendered (`renderOn`). This component was handling all the logic for how to change its styles according to where it was being used (`page` vs `modal`). This is better handled by just exposing some CSS variables and letting each parent define how to customize them -- this component shouldn't have to care about those details.